src/Storefront/Resources/views/storefront/page/product-detail/buy-widget.html.twig

```diff
@@ -50,7 +50,7 @@
 
             {% block page_product_detail_rich_snippets_release_date %}
                 <meta itemprop="releaseDate"
-                      content="{{ page.product.releaseDate|format_date('short', locale=app.request.locale) }}"/>
+                      content="{{ page.product.releaseDate|format_date(pattern="Y-MM-dd", locale=app.request.locale) }}"/>
```

Comment:
@tinect I was asking myself - Is that pattern "Y-MM-dd" correct? It looks a little bit unusual with only one 'Y'.

Reply:
@raknison Yes :-) but you can use `yyyy`, too.

Reply:
will this have a different date format for different languages?

Reply:
oh it is a meta prop, so probably it doesn't matter 🤦‍♂️